auxin_cli/src/main.rs

```diff
@@ -327,29 +349,34 @@ pub async fn main() -> Result<()> {
 			) = mpsc::channel(MESSAGE_BUF_COUNT);
 
 			tokio::task::spawn_blocking(move || {
```

Comment:
This just moves the work into the success case so I don't have to deal with possibly uninitialized values. Kinda gross indents but :shrug: